administrator/components/com_associations/models/forms/filter_associations.xml

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="utf-8"?>
+<form>
+	<field
+		name="associationcomponent"
+		type="associatedcomponent"
+		label="COM_ASSOCIATIONS_FILTER_SEARCH_LABEL"
+		description="COM_ASSOCIATIONS_FILTER_SEARCH_DESC"
+		onchange="this.form.submit();"
+		>
+		<option value="">COM_ASSOCIATIONS_FILTER_SELECT_ITEM_TYPE</option>
+	</field>
+	<field
+		name="associationlanguage"
+		type="contentlanguage"
+		label="JOPTION_FILTER_LANGUAGE"
+		description="JOPTION_FILTER_LANGUAGE_DESC"
+		onchange="this.form.submit();"
+		>
+		<option value="">JOPTION_SELECT_LANGUAGE</option>
+	</field>
+	<fields name="filter">
+		<field
+			name="search"
+			type="text"
+			label="COM_ASSOCIATIONS_FILTER_SEARCH_LABEL"
+			description="COM_ASSOCIATIONS_FILTER_SEARCH_DESC"
+			hint="JSEARCH_FILTER"
+		/>
+	</fields>
+
+	<fields name="list">
+		<field
+			name="fullordering"
+			type="list"
+			label="COM_ASSOCIATIONS_LIST_FULL_ORDERING"
+			description="COM_ASSOCIATIONS_LIST_FULL_ORDERING_DESC"
+			default="a.name ASC"
+			onchange="this.form.submit();"
+			>
+			<option value="">JGLOBAL_SORT_BY</option>
+			<option value="a.ordering ASC">JGRID_HEADING_ORDERING_ASC</option>
+			<option value="a.ordering DESC">JGRID_HEADING_ORDERING_DESC</option>
+			<option value="a.lft ASC">JGRID_HEADING_ORDERING_ASC</option>
+			<option value="a.lft DESC">JGRID_HEADING_ORDERING_DESC</option>
```

Comment:
@jreys you can't have two ordering orders

you need to do like you did for the title, i.e:
- put just `ordering` here, replace everywhere `a.ordering` for `ordering`
- and in the modal getListQuery method use `a.title AS ordering` or `a.lft AS ordering` by checkin which componet the user selected